package.json

```diff
@@ -57,40 +57,40 @@
     "watch:src": "tsc -w"
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.0.0",
-    "@jupyterlab/cells": "^3.0.0",
-    "@jupyterlab/docregistry": "^3.0.0",
-    "@jupyterlab/notebook": "^3.0.0",
-    "@jupyterlab/settingregistry": "^3.0.0",
-    "@lumino/coreutils": "^1.5.3",
-    "@lumino/widgets": "^1.16.1",
-    "date-fns": "^2.24.0"
+    "@jupyterlab/application": "^4.0.0-alpha.22",
+    "@jupyterlab/cells": "^4.0.0-alpha.22",
+    "@jupyterlab/docregistry": "^4.0.0-alpha.22",
+    "@jupyterlab/notebook": "^4.0.0-alpha.22",
+    "@jupyterlab/settingregistry": "^4.0.0-alpha.22",
+    "@lumino/coreutils": "^2.0.0-rc.1",
+    "@lumino/widgets": "^2.0.0-rc.1",
+    "date-fns": "^2.29.3"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint && npm run test"
     }
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.1.12",
-    "@types/chai": "^4.2.22",
-    "@types/mocha": "^9.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.32.0",
-    "@typescript-eslint/parser": "^4.32.0",
-    "chai": "^4.3.4",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "husky": "^4.2.3",
-    "mocha": "^9.1.2",
+    "@jupyterlab/builder": "^4.0.0-alpha.22",
+    "@types/chai": "^4.3.4",
+    "@types/mocha": "^10.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.54.1",
+    "@typescript-eslint/parser": "^5.54.1",
+    "chai": "^4.3.7",
+    "eslint": "^8.35.0",
+    "eslint-config-prettier": "^8.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "husky": "^8.0.3",
+    "mocha": "^10.2.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.4.1",
-    "rimraf": "^3.0.2",
-    "ts-mocha": "^8.0.0",
+    "prettier": "^2.8.4",
+    "rimraf": "^4.4.0",
+    "ts-mocha": "^10.0.0",
     "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "~4.1.3"
+    "typescript": "5.0.1-rc"
```

Comment:
With JupyterLab [4.0.0a37](https://github.com/jupyterlab/jupyterlab/releases/tag/v4.0.0a37) we now use TypeScript 5.0 final (https://github.com/jupyterlab/jupyterlab/pull/14215).

Let's update all versions again, including eslint (https://github.com/jupyterlab/jupyterlab/pull/14216).

Reply:
Sounds good. I've Updated these all to match jupyterlab.